mautic/mautic
# Allow admin details to be provided in local.php for CLI installer

This changes where the defaults for the admin username, firstname and 
lastname are provided, as the previous cli option defaults were 
overriding the values provided by local config (local.php or environment 
variables)

Closes issue #10018

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features 
| Bug fix?                               | yes
| Issue(s) addressed                     | Fixes #10018 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Install using the CLI installer with 'admin_username' set to 'root' in local.php 
3. auto-created admin username should be 'root'
